pages/processes/favourite.tsx

```diff
@@ -1,32 +1,35 @@
 import React, { useState } from "react";
-import styles from "./FrontPage.module.scss";
+
+import ActiveFilterSection from "components/Labels/ActiveFilterSection";
+import FilterLabelButton from "components/Labels/FilterLabelButton";
+import FilterUserButton from "components/FilterUserButton";
+import FrontPageBody from "components/FrontPageBody";
 import Head from "next/head";
 import { Layouts } from "../../layouts/LayoutWrapper";
+import { SearchField } from "components/SearchField";
 import SideNavBar from "components/SideNavBar";
-import FrontPageBody from "components/FrontPageBody";
-import { useQuery } from "react-query";
+import { SortSelect } from "../../components/SortSelect";
+import { Typography } from "@equinor/eds-core-react";
 import { getProjects } from "../../services/projectApi";
+import { stringToArray } from "utils/stringToArray";
+import styles from "./FrontPage.module.scss";
+import { useQuery } from "react-query";
 import { useRouter } from "next/router";
-import { Typography } from "@equinor/eds-core-react";
-import { SortSelect } from "../../components/SortSelect";
-import { SearchField } from "components/SearchField";
-import FilterLabelButton from "components/Labels/FilterLabelButton";
-import ActiveFilterSection from "components/Labels/ActiveFilterSection";
 
 export default function FavoriteProcesses(): JSX.Element {
   const [page, setPage] = useState(1);
   const itemsPerPage = 16;
 
   const router = useRouter();
-
   const query = useQuery(
     ["favProjects", page, "isFavourite", ...Object.values(router.query)],
     () =>
       getProjects({
         page,
         items: itemsPerPage,
         onlyFavorites: true,
-        ...router.query,
+        ru: stringToArray(router.query.user),
+        rl: stringToArray(router.query.rl),
```

Comment:
I believe this is cleaner. Let me know what you think :) @dansvend

Reply:
Perfect👌